stories/setup/styles.scss

```diff
@@ -64,7 +64,7 @@ body.fullscreen {
 .FullscreenGridStory-content,
 .FullscreenGridStory-altSidePanel {
   align-items: center;
-  display: flex;
```

Comment:
I think you can just remove `display: flex` and replace it with nothing. That seems to work locally for me. Does that not work for you?

Reply:
Oh yeah, getting rid of `display` entirely works for me as well. I'll update the PR.